TradeItIosTicketSDK2/TradeItLinkedBrokerCache.swift

```diff
@@ -0,0 +1,98 @@
+import SwiftyUserDefaults
+
+// https://github.com/radex/SwiftyUserDefaults
+
+extension DefaultsKeys {
+    static let linkedBrokerCache = DefaultsKey<NSDictionary?>("linkedBrokerCache")
+}
+
+class TradeItLinkedBrokerCache {
+    typealias UserId = String
+    typealias AccountNumber = String
+    typealias SerializedLinkedBroker = [String: AnyObject]
+    typealias SerializedLinkedBrokers = [UserId: SerializedLinkedBroker]
+    typealias SerializedLinkedBrokerAccount = [String: String]
+    typealias SerializedLinkedBrokerAccounts = [AccountNumber: SerializedLinkedBrokerAccount]
+
+    private let ACCOUNTS_KEY = "ACCOUNTS"
+    private let ACCOUNTS_LAST_UPDATED_KEY = "ACCOUNTS_LAST_UPDATED"
+    private let ACCOUNT_NAME_KEY = "ACCOUNT_NAME"
+
+    var defaults = NSUserDefaults(suiteName: "it.trade")!
+
+    func cache(linkedBroker linkedBroker: TradeItLinkedBroker) {
+        guard let userId = linkedBroker.linkedLogin.userId else { return }
+
+        var linkedBrokerCache = defaults[.linkedBrokerCache] as? SerializedLinkedBrokers ?? SerializedLinkedBrokers()
```

Comment:
What is `SerializedLinkedBrokers` doing here? Creating an empty array of that type?